Gemfile

```diff
@@ -47,8 +47,8 @@ gem 'hypdf', '~> 1.0.10'          # PDFInfoAgent
 # FIXME needs to loosen omniauth dependency
 gem 'weibo_2', github: 'dsander/weibo_2', branch: 'master'
 
-# GoogleCalendarPublishAgent
-gem "google-api-client", require: 'google/api_client'
```

Comment:
Can you add back in the `require: 'google/api_client'` so that the GoogleCalendarPublishAgent will also work ok? I think you should also copy the `gem_dependency_check` and `#{'## Include `google-api-client` in your Gemfile to use this Agent!' if dependencies_missing?}` lines from GoogleCalendarPublishAgent into your Agent so that it correctly detects if `google-api-client` has been removed from the Gemfile, and remove the `require` line that you added at the top.